Flask==2.3.3
line-bot-sdk==3.8.0 # LineBotApiErrorのImportErrorを解決するため、バージョンを3.8.0に固定
google-generativeai==0.5.0
python-dotenv
gunicorn
